appserver/connectors/work-management/src/main/java/com/sun/enterprise/connectors/work/WorkCoordinator.java

```diff
@@ -91,10 +99,10 @@
 
     private WorkManagementProbeProvider probeProvider = null;
 
-    private ConnectorRuntime runtime;
+    private final ConnectorRuntime runtime;
```

Comment:
Eclipse automatically adds final to fields that are set only in constructor and never changed. (yes, you can still call their methods).